js/src/examples/controls/OrbitControls.js

```diff
@@ -208,6 +208,23 @@ THREE.OrbitControls = function ( object, domElement ) {
 
 	}();
 
+	this.connectEvents = function(element) {
+		if (element) {
+			scope.domElement = element;
+		}
+		scope.domElement.addEventListener( 'contextmenu', onContextMenu, false );
```

Comment:
This I would actually keep, as these are changes to the vendored three.js code, and as such is in the same style as the reset of the file (it also uses tabs over spaces).